robot/web/web.go

```diff
@@ -354,7 +354,6 @@ func (svc *webService) refreshResources() error {
 }
 
 func (svc *webService) updateResources(resources map[resource.Name]resource.Resource) error {
-	// so group resources by API
 	groupedResources := make(map[resource.API]map[resource.Name]resource.Resource)
```

Comment:
I know this is existing, but I was curious -- is the `map[resource.Name]resource.Resource` bit ever being used as a map to do a name -> resource lookup? I tried scanning the code for that and don't think I saw any.

Reply:
I don't remember so, it's more to keep the name of the resource around (I guess now that all resources are Named, that may not be necessary)

Reply:
Got it, that's what I figured.